pkg/cli/cmd/credential/show/objectformats_test.go

```diff
@@ -76,23 +76,52 @@ func Test_credentialFormat_Azure_WorkloadIdentity(t *testing.T) {
 	require.Equal(t, expected, buffer.String())
 }
 
-func Test_credentialFormat_AWS(t *testing.T) {
+func Test_credentialFormat_AWS_AcessKey(t *testing.T) {
```

Comment:
We should write the whole name of the function.
```suggestion
func Test_ credentialFormatAWSAccessKey(t *testing.T) {
```